src/app/pages/auth/containers/enrolment-page.component.html

```diff
@@ -84,11 +84,22 @@
               class="bt bt--full"
               round
               full
-              (click)="enterToken()"
+              (click)="enrol('token')"
```

Comment:
This can be made less prominent by moving it to another page along with any other options (such as enter study url). In the main page here, we just say in small text "Login using other methods" or similar